scripts/shared/cloud-prepare.sh

```diff
@@ -29,7 +29,58 @@ function check_gateway_exists() {
 
 function prepare_kind() {
     read -r -a nodes <<< "${cluster_nodes[$cluster]}"
-    kubectl label node "${cluster}-${nodes[-1]}" "${GATEWAY_LABEL}" --overwrite
+
+    # In order to test all gateway fail over test scenarios,
+    # one cluster will label 1 GW node and second 2 GW nodes.
+    worker_count=$(grep -o worker <<< "${nodes[*]}" | wc -l)
```

Comment:
I think it'll be more in line with Shipyard if you added support for labeling GW nodes in the settings, this way you can specify in the settings explicitly which nodes to label if you want (and if you dont, the default can remain the last node on the list).

Reply:
By "adding support for labeling GW node in the setting" you mean to add a parameterized argument here?
https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/lib/utils#L207

Reply:
Yes, specifically in this loop https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/lib/utils#L227

Reply:
Implemented